bot/handlers.py

```diff
@@ -78,66 +87,146 @@ def handle_team_settings_commands(
         scheduler: AsyncIOScheduler, send_message: SendMessage, router: Router, bot: Bot
 ):
     @router.message(
-        Command(bot_command_names.set_meetings_time), HasMessageText(), HasChatState()
+        Command(bot_command_names.set_meetings_time_zone), HasMessageText(), HasChatState()
     )
-    async def set_meetings_time(
+    async def set_meetings_time_zone(
             message: Message, message_text: str, chat_state: ChatState
     ):
-        meeting_time_str = message_text.split(" ", 1)
-        topic_id = message.message_thread_id
-
         try:
-            meeting_time = datetime.fromisoformat(meeting_time_str[1])
-            chat_state.meeting_time = meeting_time
-            chat_state.topic_id = topic_id
-            await save_state(chat_state=chat_state)
+            argument = message_text.split(" ")[1]
 
-            schedule_meeting(
-                meeting_time=meeting_time,
-                chat_id=chat_state.chat_id,
-                scheduler=scheduler,
-                send_message=send_message,
-            )
+            try:
+                if ":" not in argument:
+                    raise IndexError
 
-            username = message.from_user.username if message.from_user else None
+                hour = int(argument.split(":")[0])
+                minute = int(argument.split(":")[1])
 
-            await update_reminders(
-                bot=bot,
-                username=username,
-                scheduler=scheduler,
-                send_message=send_message
-            )
+                timezones = guess_time_zone(hour, minute)
+
+                if timezones:
+                    await message.reply(
+                        text="Chose your time zone",
+                        reply_markup=make_timezones_inline_keybard(timezones)
+                    )
+                else:
+                    await message.reply("Sorry, no time zones found")
+
+                return
+            except IndexError:
```

Comment:
Why do you need an `IndexError`? It seems like you can use just `if`-s without `try .. except`.